.codeclimate.yml

```diff
@@ -6,7 +6,7 @@ engines:
       languages:
       - javascript
   eslint:
-    enabled: true
+    enabled: false
```

Comment:
CodeClimate kept freaking out about this and I got annoyed. This is a `wontfix` for now